src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -92,8 +91,9 @@
     );
 
     /** Contains the verified itinerary set for a trip upon persisting. */
-    private static final Itinerary DEFAULT_ITINERARY =
-        OtpTestUtils.OTP_DISPATCHER_PLAN_RESPONSE.getResponse().plan.itineraries.get(0);
+    public static Itinerary getDefaultItinerary() throws Exception {
```

Comment:
Why not the more precise `JsonProcessingException`?

Reply:
Cause I was lazy and this is in testing code.